z3c/dependencychecker/package.py

```diff
@@ -28,21 +34,26 @@ def __init__(self, path):
 
     @cached_property
     def distribution_root(self):
-        if "setup.py" not in os.listdir(self._path):
+        for metadata_file in METADATA_FILES:
+            if metadata_file in os.listdir(self._path):
+                break
+        else:
             logger.error(
-                "setup.py was not found in %s. "
+                "No distribution configuration file was not found in %s. "
```

Comment:
"Distribution configuration file" sounds a bit abstract. `pyproject.toml/setup.py/.cfg` is more explicit (and shorter).